pyproject.toml

```diff
@@ -41,17 +44,21 @@ hvac = "~0.11.0"  # (always keep in sync w/ optional dependency)
 invoke = "*"
 moto = "~2.2.11"
 mkdocs = "*"
+# Pin this to the lowest supported version of Nautobot
+nautobot = "~1.2.1"
```

Comment:
This shouldn't need to be added to `dev-dependencies` as it's already a core dependency of the project (line 22 above).

Reply:
Updated pyproject and fixed pydocstyle, flake8